components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/DateTimePicker.jsx

```diff
@@ -238,21 +236,26 @@ export default class DateTimePicker extends React.Component {
                                 <br />
                                 Time
                                 <br />
-                                <div
-                                    className="MuiFormControl-root-69"
-                                >
-                                    <div
-                                        className='MuiInput-root-52 MuiInput-formControl
-                                        -53 MuiInput-underline-56 underline'
-                                    >
+                                <div>
+                                    <div>
                                         <input
-                                            className="MuiInput-input-60"
                                             type='time'
                                             step={this.getTimeStep(inputType)}
                                             value={time}
                                             onChange={(evt) => {
                                                 this.handleOnChange('time', evt.target.value);
                                             }}
+                                            style={{
+                                                color: theme.palette.textColor,
+                                                backgroundColor: theme.palette.canvasColor,
+                                                borderBottom: '1px solid',
+                                                borderTop: 'none',
+                                                borderLeft: 'none',
+                                                borderRight: 'none',
+                                                width: 200,
```

Comment:
Hard-coding width, height in `px` is not a good practice. Is there a way to set these as percentages?

Reply:
This content is rendered inside a fixed size popover, therefore using percentages isn't needed as the elements are not expected to resize. That's why px is used.